docs/developer-docs/latest/setup-deployment-guides/deployment/strapi-cloud.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Strapi Cloud Deployment - Strapi Developer Docs
+description: Learn how to deploy your Strapi application on Strapi Cloud.
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/setup-deployment-guides/deployment/strapi-cloud.html
+---
+
+# Strapi Cloud
+
+::: warning
+Strapi Cloud is currently in Closed Beta release. Features and functionality may change prior to General Availability.
+
+Sign up for the waitlist [here](https://strapi.io/cloud).
+:::
+
+This is a step-by-step guide for deploying your Strapi application on Strapi Cloud.
+
+## Prerequisites
+
+Before you can deploy your Strapi application on Strapi Cloud, you need to have the following prerequisites:
+
+* A [GitHub](https://github.com) account
+* GitHub repository for each Strapi application to be deployed
+
+::: tip
+Each Strapi app must be in a separate repository. Your Strapi Cloud account can link to multiple repositories to deploy multiple apps, if needed.
```

Comment:
Next sprint we will work on [this US](https://strapi-inc.atlassian.net/browse/CLOUD-128).
So it would be possible to have multiple Strapi apps in one repo, and users will be able to select a particular sub-folder with a Strapi app inside certain repo. Maybe we should we reword this tip then, wdyt ?